serialport.js

```diff
@@ -278,12 +278,12 @@ function SerialPortFactory(_spfOptions) {
   };
 
   SerialPort.prototype.isOpen = function() {
-    return (this.fd ? true : false);
+    return (this.fd !== null ? true : false);
```

Comment:
Yes, indeed, I agree. I only tried to minimize the patch.

I just rewrote that commit